code/modules/clothing/suits/cloaks.dm

```diff
@@ -89,3 +89,41 @@
 	heat_protection = HEAD
 	max_heat_protection_temperature = FIRE_IMMUNITY_MAX_TEMP_PROTECT
 	resistance_flags = FIRE_PROOF | ACID_PROOF
+
+/obj/item/clothing/neck/cloak/skill_reward
+	var/associated_skill_path
+	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | 
+
+/obj/item/clothing/neck/cloak/skill_reward/examine(mob/user)
+	. = ..()
+	. += "<span class='notice'>You notice a powerful aura about this cloak, suggesting that only the truly experienced may wield it.</span>"
+
+/obj/item/clothing/neck/cloak/skill_reward/equipped(mob/user, slot)
+	..()
+	if (user.mind?.skill_experience[GetSkillRef(associated_skill_path)] < SKILL_EXP_LEGENDARY)
+		to_chat(user, "<span class = 'notice'>You feel completely and utterly unworthy to even hold \the [src]. You set it on the ground out of respect.</span>")
+		user.dropItemToGround(src, TRUE)
+
+/obj/item/clothing/neck/cloak/skill_reward/gaming
+	name = "legendary gamer's cloak"
+	desc = "Worn by the most skilled professional gamers on the station, this legendary cloak is only attainable by achieving true gaming enlightenment. This status symbol represents the awesome might of a being of focus, commitment, and sheer fucking will. Something casual gamers will never begin to understand."
+	icon_state = "gamercloak"
+	associated_skill_path = /datum/skill/gaming
+
+/obj/item/clothing/neck/cloak/skill_reward/cleaning
+	name = "legendary cleaner's cloak"
+	desc = "Worn by the most skilled of custodians, this legendary cloak is only attainable by achieving janitorial enlightenment. This status symbol represents a being not only extensively trained in grime combat, but one who is willing to use an entire aresenal of cleaning supplies to its full extent to wipe grime's miserable ass off the face of the station."
+	icon_state = "cleanercloak"
+	associated_skill_path = /datum/skill/cleaning
+
+/obj/item/clothing/neck/cloak/skill_reward/healing
+	name = "legendary healer's cloak"
+	desc = "Worn by the most skilled healers, this legendary cloak is only attainable by achieving true medical enlightenment. This status symbol represents a being who has saved enough lives to repopulate a small country, a being who could transplant a monkey's brain into your skull faster than you could yell ;HELP SEC."
```

Comment:
end is a bit OOC

Reply:
yeah but its funny and a rare enough description